flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergStreamWriter.java

```diff
@@ -230,10 +230,34 @@ public void testBoundedStreamCloseWithEmittingDataFiles() throws Exception {
 
       result = WriteResult.builder().addAll(testHarness.extractOutputValues()).build();
       Assert.assertEquals(0, result.deleteFiles().length);
-      Assert.assertEquals(expectedDataFiles * 2, result.dataFiles().length);
+      // Datafiles should not be sent again.
+      Assert.assertEquals(expectedDataFiles , result.dataFiles().length);
     }
   }
+  @Test
```

Comment:
need a new line btw test methods

Reply:
I'm not sure I understand what you mean. What kind of approach is needed?